source/compute/portal/ondemand.rst

```diff
@@ -422,10 +422,12 @@ RStudio Server
 --------------
 
 This interactive app allows you to run an RStudio session on the cluster.
-In the 'Toolchain year and R version' drop-down menu, you can choose the version
-of R module that would be loaded for your session (such as ``R/4.2.2-foss-2022b``).
-Additionally, the ``R-bundle-CRAN`` and ``R-bundle-Bioconductor`` modules can be loaded
-on top of the base R module to provide easy access to hundreds of preinstalled packages.
+In the 'R version' drop-down menu, you can choose the version
+of R module that would be loaded for your session (such as ``R/4.4.1-foss-2023b``).
```

Comment:
Users will get confused when they start searching for a `R/4.4.1-foss-2023b` module and find that it doesn't exist. (Because what happens when selecting `R/4.4.1-foss-2023b` is that `RStudio-Server/2024.09.0+375-foss-2023b-Java-11-R-4.4.1` and `R-bundle-CRAN/2024.06-foss-2023b` modules get loaded, which load `R/4.4.1-gfbf-2023b` as a dependency.)

Reply:
oops. my bad. I will fix the drop-down menu in OOD and the text here.